client/app/pages/queries/view.js

```diff
@@ -305,6 +305,7 @@ function QueryViewCtrl(
     Events.record('edit_name', 'query', $scope.query.id);
     if ($scope.query.is_draft && clientConfig.autoPublishNamedQueries && $scope.query.name !== 'New Query') {
       $scope.query.is_draft = false;
+      notification.success('Query published');
```

Comment:
This will result in multiple notifications + might show a success notification while the query save actually fails. 

How about we update the save notification to include a message about publishing the query? We can do a check in the `saveQuery` function to see if the publish state changed and update the message accordingly.

Reply:
Done 997182b 👍 

<img width="365" alt="Screen Shot 2019-07-20 at 22 23 13" src="https://user-images.githubusercontent.com/486954/61583182-3c07a700-ab3d-11e9-87a3-1d13eacb170f.png">